README.md

```diff
@@ -11,35 +11,24 @@ The package provides 2 sets of rules:
 
 Copy `.prettierrc` file into root of your project. Make sure that you do not change configuration or you will run in a bunch of conflicts between prettier and eslint.
 
-## To install base `eko` rules set:
+## Installation:
 
-### 1. Run npm insall for all required packages and `eslint-config-eko`:
+### 1. Install `eslint-config-eko` and peer-dependencies:
 
-`npm i --save-dev eslint prettier eslint-config-airbnb-base eslint-config-prettier eslint-plugin-import eslint-plugin-prettier eslint-config-eko`
+(mandatory: base) `npx install-peerdeps --dev -x "--save-prefix=~" eslint-config-eko@2.0.0`
```

Comment:
I've run this command in a new repository and nothing gets installed. Am I doing something wrong or is it broken?

Reply:
eslint-config-eko@2.0.0 is not release yet. 
you can try with `eslint-config-eko@develop` or `eslint-config-eko@2.0.0-rc2`